Xamarin.Forms.Core/Internals/NotifyCollectionChangedEventArgsEx.cs

```diff
@@ -1,9 +1,9 @@
 using System.Collections;
 using System.Collections.Specialized;
 
-namespace Xamarin.Forms
+namespace Xamarin.Forms.Internals
 {
-	internal class NotifyCollectionChangedEventArgsEx : NotifyCollectionChangedEventArgs
```

Comment:
Anyone know why we need NotifyCollectionChangedEventArgsEx? Seems we could compute the Count of items changed using an extension method.. Not that it really matters as this is not part of our official public surface area.